Library/Formula/sfml.rb

```diff
@@ -27,9 +17,41 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
 
   def install
-    system "cmake", ".", "-DCMAKE_INSTALL_FRAMEWORK_PREFIX=#{frameworks}/", *std_cmake_args
+    args = std_cmake_args
+    args << "."
+
+    args << "-DSFML_BUILD_DOC=TRUE" if build.with? "doxygen"
+
+    # If we leave this dir present, sfml will always prefer to use its bundled deps
+    # instead of ours, and that breaks absolutely everything.
+    rm_rf buildpath/"extlibs"
```

Comment:
(But generally, having to specify a large amount of variables to CMake is extremely prone to breakage, because it's hard to detect when they no longer apply, why they no longer apply, etc).